examples/player/simid_player.html

```diff
@@ -132,22 +138,38 @@
       <tr>
         <td>Ad type</td>
         <td>
-          <input type="radio" id="linear_ad" name="ad_type" checked> <label for="linear_ad" class="ad_type_label">Linear
+          <input type="radio" id="linear_ad" name="ad_type" onclick="toggleInputs()" checked> <label for="linear_ad" class="ad_type_label">Linear
```

Comment:
If one of these buttons is already checked, but you click it again anyway, does this trigger the toggleInputs function?

IMO it would be safer to structure this more similar to how the setCreativeControlState function works, so it takes in the value for which it wants to set the button (https://github.com/InteractiveAdvertisingBureau/SIMID/blob/master/examples/player/simid_player.js#L419).

Toggle is good when the meaning of the actual button is to toggle something (something like a play/pause button, where the actual meaning of the button is "change the state from what it currently is").  In this case, you want a specific outcome when clicking, regardless of what the current state is, so probably best to encode that specifically.